qiskit_bot/release_process.py

```diff
@@ -30,31 +29,6 @@
 LOG = logging.getLogger(__name__)
 
 
-def _regenerate_authors(repo):
-    try:
-        LOG.info('Regenerating authors list')
-        res = subprocess.run(['python', 'tools/generate_authors.py'],
-                             capture_output=True, check=True,
-                             cwd=repo.local_path)
-        LOG.debug('generate_authors called\nstdout:\n%s\nstderr:\n%s' % (
-            res.stdout, res.stderr))
-    except subprocess.CalledProcessError as e:
-        LOG.exception("Failed to generate_authors\nstdout:\n%s\nstderr:\n%s"
-                      % (e.stdout, e.stderr))
-        return
-    try:
-        LOG.info('Regenerating bibtex file')
-        res = subprocess.run(['python', 'tools/generate_bibtex.py'],
```

Comment:
Are the files `tools/generate_authors.py` and `tools/generate_bibtex.py` still needed after this change?

Reply:
They're not, https://github.com/Qiskit/qiskit/pull/1693 is deleting them

Reply:
Oh my bad, I thought they were in this repo sorry.

Reply:
Those are removed in Qiskit/qiskit#1693 😄 Unless they also live in another place I didn't see them 🤔